meshnets/utils/model_training.py

```diff
@@ -62,12 +61,59 @@ def compute_train_stats(dataset, max_examples=1000):
     }
 
 
+def make_dataloader(version,
+                    split_percentage,
+                    split_start,
+                    num_proc,
+                    writer_batch_size,
+                    batch_size,
+                    num_workers_loader,
+                    shuffle=True):
+    """Make a dataloader for the wind tunnel dataset."""
+    if split_start == 'beginning':
+        split = f'train[:{split_percentage:.0%}]'
+    elif split_start == 'end':
+        split = f'train[-{split_percentage:.0%}:]'
+    else:
+        raise ValueError(f'Invalid split_start: {split_start}')
+    dataset = datasets.load_dataset('inductiva/wind_tunnel',
+                                    version=version,
+                                    split=split,
+                                    download_mode='force_redownload')
```

Comment:
Why this forcing?

Reply:
To prevent caching issues. Because I have several different versions of the dataset that require different downloads I 'force_download'. Why? because, otherwise HuggingFace will detect `inductiva/wind_tunnel` in `/home/<user>/.cache/huggingface/datasets` and not run the download again. This will cause it to skip the unpack step and may cause it to work with the wrong files